JASP-Desktop/widgets/listmodeltermsassigned.cpp

```diff
@@ -127,6 +127,24 @@ const Terms &ListModelTermsAssigned::terms(const QString &what) const
 		return ListModelAssignedInterface::terms(what);
 }
 
+void ListModelTermsAssigned::removeTerm(int index)
+{
+	if (index < 0 || index >= int(_terms.size())) return
+	_tempTermsToRemove.clear();
+
+	beginResetModel();
+
+	const Term& term = _terms.at(size_t(index));
+
+	_rowControlsMap.remove(term.asQString());
+	_terms.remove(term);
+	_tempTermsToRemove.add(term);
+
+	endResetModel();
```

Comment:
If I test your branch on linux JASP crashes right here when I remove the hypothesis.

Reply:
Somewhere in qt code with setParentItem...

Reply:
There are some other problems Im having with JASP on linux lately, so Ill just go try this on windows.

Reply:
ok on windows this works, I assume I  just broke my linux install for now.